services/subscription-service/package.json

```diff
@@ -79,28 +79,28 @@
         "@opentelemetry/plugin-pg-pool": "^0.15.0",
         "@opentelemetry/sdk-trace-base": "^1.15.0",
         "@opentelemetry/sdk-trace-node": "^1.15.0",
-        "@sourceloop/core": "^16.2.0",
-        "@sourceloop/feature-toggle-service": "^4.0.2",
+        "@sourceloop/core": "^20.0.0",
```

Comment:
are all the breaking changes done in core and feature-toggle added here 
if required